src/OpenSearch.Client.JsonNetSerializer/OpenSearch.Client.JsonNetSerializer.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
Are you also deleting the OSC.JsonNetSerializer.csproj?

Reply:
Yup. In cca72917bac37c1cb05c5999440e94cfa6a2b0a0.